joining-table-table/kstreams/README.md

```diff
@@ -0,0 +1,52 @@
+<!-- title: How to join a KTable and a KTable in Kafka Streams -->
+<!-- description: In this tutorial, learn how to join a KTable and a KTable in Kafka Streams, with step-by-step instructions and supporting code. -->
+
+# How to join a KStream and a KTable in Kafka Streams
+
+Suppose you have a set of movies that have been released and a stream of ratings from moviegoers about how entertaining they are.  But you're only interested in the latest rating and since the `KTable` is an update stream, you'll want to use a `KTable` for the ratings.   In this tutorial, we'll write a program that joins the latest rating with content about the movie.
+
+First you'll create a `KTable` for the reference movie data
+```java
+KTable<Long, Movie> movieTable = builder.stream(MOVIE_INPUT_TOPIC,
+                Consumed.with(Serdes.Long(), movieSerde))
+        .peek((key, value) -> LOG.info("Incoming movies key[{}] value[{}]", key, value))
+        .toTable(Materialized.with(Serdes.Long(), movieSerde));
+```
+Here you've started with a `KStream` so you can add a `peek` statement to view the incoming records, then convert it to a table with the `toTable` operator.  Otherwise, you could create the table directly with `StreamBuilder.table`.  Here we are assuming the underlying topic is keyed with the movie id.
```

Comment:
```suggestion
Here you've started with a `KStream` so you can add a `peek` statement to view the incoming records, then convert it to a table with the `toTable` operator.  Otherwise, you could create the table directly with `StreamBuilder.table`. We assume that the underlying topic is keyed on the movie ID.
```